modules/setting/time.go

```diff
@@ -9,45 +9,10 @@ import (
 	"code.gitea.io/gitea/modules/log"
 )
 
-var (
-	// Time settings
-	TimeFormat string
-	// UILocation is the location on the UI, so that we can display the time on UI.
-	DefaultUILocation = time.Local
-)
+// DefaultUILocation is the location on the UI, so that we can display the time on UI.
+var DefaultUILocation = time.Local
```

Comment:
Can we get rid of this as well? Where's it used?

Reply:
It's used in a few places in the codebase so not as straightforward as `FORMAT` but I think that it's probably irrelevant as well

Reply:
Yes, it seems nonsensical for the server to try to guess user location. It will only work if all users are in the same timezone. Let's drop it in another PR.